Assets/MRTK/Tests/PlayModeTests/BaseCursorTests.cs

```diff
@@ -33,7 +33,7 @@ public class BaseCursorTests
         [SetUp]
         public void SetUp()
         {
-            TestUtilities.InitializeMixedRealityToolkit(true);
+            PlayModeTestUtilities.Setup();
```

Comment:
Now that input simulation is getting more complex, it would be nice to add a test to verify that input simulation is working as intended, that is when you left click a single select event is raised, when hand is up and you click a single event is raised and not two events (one from the hand, and one from the gaze cursor), and so on. Since it might be difficult to directly simulate pressing keys and clicking mouse buttons, you could write methods to the input simulation service to expose things like "press left hand control key" or "press left hand activate button", "Press left hand toggle key", etc.

This could be done as part of this PR, or if you'd like to split it into two PRs, please file a bug and address it in a follow-up PR. Note that it would be good to get both the feature and test tests in the same release, so that we can be confident future work doesn't break existing functionality. It will also help increase confidence that this change doesn't have unwanted side effects.

Reply:
Looking through posts about how other developers have handled simulating input I think a larger scale rewrite would have to be done to properly test an end user's experience

https://answers.unity.com/questions/28473/simulating-key-presses.html
https://answers.unity.com/questions/588786/is-it-possible-to-simulate-a-keypressinput-in-code.html

For now I'll write a set of tests explicitly checking the new hands-free ggv feature and log a broader work item for later

Reply:
I actually found that we have a KeyInputSystem class that we can leverage to do input simulation, going to modify that class to allow for simulated inputs